packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -644,27 +647,28 @@ export type SdkHttpParameter =
   | SdkPathParameter
   | SdkBodyParameter
   | SdkHeaderParameter
-  | SdkCookieParameter;
+  | SdkCookieParameter
+  | SdkServiceResponseHeader;
```

Comment:
why is `SdkServiceResponseHeader` in `SdkHttpParameter`?

Reply:
removed.